docker-app/qfieldcloud/settings.py

```diff
@@ -234,7 +234,7 @@
 
 # QFieldCloud variables
 AUTH_TOKEN_LENGTH = 100
-AUTH_TOKEN_EXPIRATION_HOURS = 24 * 30
+AUTH_TOKEN_EXPIRATION_HOURS = int(os.environ.get("QFIELDCLOUD_AUTH_TOKEN_EXPIRATION_HOURS", 24 * 30))
```

Comment:
```suggestion
AUTH_TOKEN_EXPIRATION_HOURS = int(
    os.environ.get("QFIELDCLOUD_AUTH_TOKEN_EXPIRATION_HOURS", 24 * 30)
)
```
Should I apply this patch to fix the test? I wouldn't because in online it looks nicer to me. `DEBUG` doesn't meet this too.

Reply:
It's not a matter of taste, it's a matter of tooling. Please apply the patch to get this merge-able.